salt/modules/suse_apache.py

```diff
@@ -23,7 +23,7 @@ def __virtual__():
     '''
     Only load the module if apache is installed.
     '''
-    if salt.utils.path.which('apache2ctl') and __grains__['os_family'] == 'SUSE':
+    if salt.utils.path.which('apache2ctl') and __grains__['os_family'].upper() == 'SUSE':
```

Comment:
This is a wrong fix. It has to look for `Suse`, not modifying anything just because it is a module, like so:
```python
.... and __grains__['os_family'] == 'Suse':
```